### PR TITLE
VAN-2821 Test pipeline added

### PR DESCRIPTION
**Pull request by rajatsharma111**
In this pr-
1- Feature branch updated to master/main in the bundles
2- Test pipeline added for pr/commit validation

### CODE REVIEW

File: wpdemo/bundle/aws/eks/app.yml [hunk 1]
```diff
@@ -9,8 +9,9 @@ application:
     - name: wordpress_container_image
       id: 1435bc98-bea6-11eb-8529-0242ac130003
       containerImage:
-        provider: custom_registry
-        repo: wordpress
+        provider: dockerhub_container_registry
+        host: index.docker.io
```
Comment (dshelley66) [inline wpdemo/bundle/aws/eks/app.yml:13]: I don't believe this is required

File: wpdemo/bundle/aws/eks/env.yml [hunk 2]
```diff
@@ -7,7 +7,7 @@ environment:
     driver: github
     repo: https://github.com/cldcvr/codepipes-tutorials
     dir: wpdemo/infra/aws/eks
-    identifier: main
+    identifier: rajat/VAN-2821
```
Comment (dshelley66) [inline wpdemo/bundle/aws/eks/env.yml:10]: should be removed before merge?

Comment (rajatsharma111) [reply wpdemo/bundle/aws/eks/env.yml:10]: yes we can @dshelley66 but the pipeline will fail with 1 test. Should we run the pipeline or can ignore it for this time?

File: wpdemo/bundle/aws/eks/app.yml (hunk 1 above)
Comment (rajatsharma111) [reply wpdemo/bundle/aws/eks/app.yml:13]: removed

File: wpdemo/bundle/aws/eks/env.yml (hunk 2 above)
Comment (rajatsharma111) [reply wpdemo/bundle/aws/eks/env.yml]: @dshelley66 @kanaksinghal @shwetanshu-cldcvr I have incorporated the review comments and the pipeline is failing with 1 test as expected.

[Result Link](https://storage.cloud.google.com/vg-api-test-report/automation/api/results/13-Jul-2022/14:23:08)

**Review by kanaksinghal — APPROVED** [review]

File: wpdemo/bundle/aws/eks/env.yml (hunk 2 above)
Comment (dshelley66) [reply wpdemo/bundle/aws/eks/env.yml]: do we have a fix queued up for the failing test? What is the jira for it?

Comment (dshelley66) [reply wpdemo/bundle/aws/eks/env.yml]: ah ok

**Review by dshelley66 — APPROVED** [review]

**Pull request merged**